src/components/Header.jsx

```diff
@@ -97,8 +97,9 @@ const Header = ({
         bg="white"
         h="4rem"
         w="100%"
+        spacing="0.5rem"
       >
-        <HStack spacing="1.25rem" flex={1}>
+        <HStack spacing="1.25rem" flex="1 1 content">
```

Comment:
this means that the `HStack` will have space to grow! we might not want this - we want all the available space to go to the title instead. should this be `0 0 content` instead? 

this means that it has enough space to fit the content and won't grow/shink. (if you feel that it should shrink then feel free to change)

Reply:
Ah ok! Removing the grow, tysm

If we don't set `flex-shrink` for all of the contents in the stack tho, how do we prevent the header contents from exceeding the width like below?

![image](https://github.com/isomerpages/isomercms-frontend/assets/139780851/905a6531-bac1-42b0-a0b5-75dac4cdff89)

Reply:
Will remove the growth factor! I think it should shrink just in case the text is rlly long!

> btw, i also think we set noOfLines = {1} on the back button text so it truncates nicely - should be also be doing it here (eg: we access a page on a subfolder iwth a very long name)

Also added this